CodingPractice/PythonAssignments/tests/test_UniqueCustomerName.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from CodingPractice.PythonAssignments.excercises.UniqueCustomerName import unique_customer_name
+
+
+class UniqueFirstNameTests(unittest.TestCase):
+
+    def test_00_unique_first_names(self):
+        name_list = [ ('Alice', 'Wonderlund') , ('Sadie', 'Frost'), ('Sam', 'Antha'), ('Sadie', 'Lady') ]
+        self.assertEqual(3, unique_customer_name(name_list))
+
+    def test_01_unique_names_with_lower_and_upper_cases(self):
```

Comment:
Tailor the test data so that it is easy to see this test is focussed on a specific case 
e.g. name_list = [('Alice', 'Wonderlund'), ('alice', 'Joy')]